compressible_flow/Transitional_Flat_Plate/LM/NLF/transitional_LM_model_ConfigFile.cfg

```diff
@@ -0,0 +1,317 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: LM model for NLF-0416 airfoil ___________________________  %
+% Author: Sunoh. Kang _______________________________________________________  %
+% Institution: Pusan National University ____________________________________  %
+% Date: 10. 24. 2022                                                           %
+% File Version 7.4.0 "Blackbird"                                               %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+% Solver type (EULER, NAVIER_STOKES, RANS,
+%              INC_EULER, INC_NAVIER_STOKES, INC_RANS,
+%              NEMO_EULER, NEMO_NAVIER_STOKES,
+%              FEM_EULER, FEM_NAVIER_STOKES, FEM_RANS, FEM_LES,
+%              HEAT_EQUATION_FVM, ELASTICITY)
+SOLVER= RANS 
+%
+KIND_TURB_MODEL= SST
+SST_OPTIONS= NONE
+%
+KIND_TRANS_MODEL= LM
+%
+% Mathematical problem (DIRECT, CONTINUOUS_ADJOINT, DISCRETE_ADJOINT)
+% Defaults to DISCRETE_ADJOINT for the SU2_*_AD codes, and to DIRECT otherwise.
+MATH_PROBLEM= DIRECT
+%
+% Restart solution (NO, YES)
+RESTART_SOL= NO
+% United States customary units (US): ( inches, slug, Rankines, lbf = slug ft/s^2,
+%                                       psf = lbf/ft^2, Density = slug/ft^3,
+%                                       Speed = ft/s, Equiv. Area = ft^2 )
+SYSTEM_MEASUREMENTS= SI
+%
+% ------------------------------- SOLVER CONTROL ------------------------------%
+%
+% Maximum number of inner iterations
+INNER_ITER= 50000
```

Comment:
All the comments please, the entire file... not just the lines I highlighted. Keep only what is essential for this feature.
Comments are very hard to maintain because their validity is not checked anywhere, therefore we do not want to repeat comments, we have the explanation of the options in the config template, and on the website.